modules/persistent-storage-hostpath-mounting.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-hostpath.adoc
+
+[id="persistent-storage-hostpath-mounting_{context}"]
+= Mounting with hostPath
+
+{product-title} supports hostPath mounting for development and testing on a single-node cluster.
+
+In a production cluster, you would not use hostPath. Instead, a cluster administrator would provision a network resource, such as a GCE Persistent Disk volume, an NFS share, or an Amazon EBS volume.
+
+Cluster administrators can also use StorageClasses to set up dynamic provisioning.
```

Comment:
Do we need to show how to set up the storage class for this?

Reply:
@sferich888 I'm investigating. hostPath dynamic provisioning is not supported, so my understanding is that storage class may not be needed at all.